projects/github-actions/repo-gardening/src/tasks/triage-issues/index.js

```diff
@@ -66,11 +66,21 @@ ${ repoLabels
 	.join( '\n' ) }
 
 Analyze the issue and suggest relevant labels. Rules:
+# Updated prompt: Ensures 'Product', 'Platform' and 'Type' issues are always added.
 - Use only existing labels provided.
+- Include 1 '[Product]' label.
```

Comment:
This seems like it will confuse the AI, since no "Product" labels are included in the list above.